src/apps/loan-list/materials/stackable-material/material-info.tsx

```diff
@@ -53,7 +53,16 @@ const MaterialInfo: FC<MaterialInfoProps> = ({
         </div>
         <div className="list-reservation__about">
           <button
-            onClick={() => openDetailsModal()}
+            onClick={(e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+              e.stopPropagation();
+              openDetailsModal();
+            }}
+            onKeyUp={(e) => {
+              if (e.key === "Enter" || e.key === "Space") {
+                e.stopPropagation();
+                openDetailsModal();
+              }
+            }}
```

Comment:
I would create a new function like

```
handleDetailsModal(event: React.SyntheticEvent) {
       event.stopPropagation();
       openDetailsModal();
}
```
And call it both places

Its a minor thing, so only if you got the time